client/src/app/pages/applications/components/application-detail-drawer/application-detail-drawer.tsx

```diff
@@ -303,6 +310,12 @@ const TabDetailsContent: React.FC<{
         onEditClick={onEditClick}
         onCloseClick={onCloseClick}
       />
+      <TextContent className={spacing.mtLg}>
+        <Title headingLevel="h3" size="md">
+          {t("terms.targets")}
+        </Title>
+        {targets?.map((t) => <Text component="small">{t}</Text>)}
```

Comment:
Need to add a `key` attribute to each mapped component.

Reply:
Also need to add a "Not available" or similar when no current analysis is available.

Reply:
I added the requested changes.